examples/website/i3s/app.js

```diff
@@ -118,21 +122,20 @@ export default class App extends PureComponent {
    * @returns {string[]} Sublayer urls or tileset url.
    * TODO Add filtration mode for sublayers which were selected by user.
    */
-  async getLayerUrls(tilesetUrl) {
+  async getFlattenedSublayers(tilesetUrl) {
     try {
       const tileset = await load(tilesetUrl, I3SBuildingSceneLayerLoader);
       const sublayersTree = buildSublayersTree(tileset.header.sublayers);
       this.setState({sublayers: sublayersTree.sublayers});
-      const urls = tileset?.sublayers
-        .map((sublayer) => sublayer.url)
-        .filter(
-          (url) =>
-            url !==
-            'https://tiles.arcgis.com/tiles/z2tnIkrLQ2BRzr6P/arcgis/rest/services/Admin_Building_v17/SceneServer/layers/0/sublayers/0'
-        );
-      return urls;
+      const sublayers = tileset?.sublayers
+        .filter((sublayer) => sublayer.name !== 'Overview')
+        .map((sublayer) => {
+          sublayer.show = true;
```

Comment:
Some sublayers it tileset are switched off by default. I don't think we should enable them all as well.

Reply:
Done